yrs/src/update.rs

```diff
@@ -333,17 +333,21 @@ impl Update {
         client_block_ref_ids: &'a mut Vec<ClientID>,
         blocks: &'b mut UpdateBlocks,
     ) -> Option<(ClientID, &'b mut VecDeque<BlockCarrier>)> {
-        loop {
-            if let Some((id, Some(client_blocks))) = client_block_ref_ids
```

Comment:
This loop would break if the poped client ID was not found on local block store. Breaking loop here means that all remaining client IDs would be dropped and never integrated.